bundles/org.openhab.binding.groupepsa/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,559 @@
+			<channel id="locked" typeId="doorLock"/>
+			<channel id="driver" typeId="doorOpen">
+				<label>Driver door</label>
+				<description>Driver door open state</description>
+			</channel>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="environment">
+		<label>Environment</label>
+		<description>Environmental conditions</description>
+		<category>Sun_Clouds</category>
+		<channels>
+			<channel id="temperature" typeId="temperature">
+				<label>Outside Temperature</label>
+				<description>Outside Temperature</description>
+			</channel>
+			<channel id="daytime" typeId="daytime"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="motion">
+		<label>Motion</label>
+		<description>Motion information</description>
+		<category>Motion</category>
+		<channels>
+			<channel id="ignition" typeId="ignition"/>
+			<channel id="acceleration" typeId="acceleration"/>
+			<channel id="moving" typeId="moving"/>
+			<channel id="speed" typeId="speed"/>
+			<channel id="mileage" typeId="mileage"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="position">
+		<label>Position</label>
+		<description>Location information</description>
+		<category>MoveControl</category>
+		<channels>
+			<channel id="position" typeId="position"/>
+			<channel id="heading" typeId="heading"/>
+			<channel id="type" typeId="type"/>
+			<channel id="signal" typeId="signal"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="various">
+		<label>Various</label>
+		<description>Miscellaneous information</description>
+		<channels>
+			<channel id="lastUpdated" typeId="lastUpdated"/>
+			<channel id="privacy" typeId="privacy"/>
+			<channel id="belt" typeId="belt"/>
+			<channel id="emergency" typeId="emergency"/>
+			<channel id="service" typeId="service"/>
+			<channel id="preconditioning" typeId="preconditioning"/>
+			<channel id="preconditioningFailure" typeId="preconditioningFailure"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="current">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Current</label>
+		<description>Electrical current</description>
+		<category>Energy</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage</label>
+		<description>Voltage</description>
+		<category>Energy</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Temperature</description>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%"/>
+	</channel-type>
+
+	<channel-type id="daytime">
+		<item-type>Contact</item-type>
+		<label>Daytime</label>
+		<description>Enabled if it is daytime</description>
+		<category>Sun</category>
+	</channel-type>
+
+	<channel-type id="doorLock">
+		<item-type>String</item-type>
+		<label>Door lock</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions